--- all/puppet-2.7.13/lib/puppet/provider/package/portage.rb-orig	2012-05-04 13:10:34.956867698 +0200
+++ all/puppet-2.7.13/lib/puppet/provider/package/portage.rb	2012-05-04 13:12:11.990194600 +0200
@@ -18,9 +18,10 @@
 
     version_format = "{last}<version>{}"
     search_format = "<category> <name> [<installedversions:LASTVERSION>] [<bestversion:LASTVERSION>] <homepage> <description>\n"
+    eix_cachefile = (eix "--print", "EIX_CACHEFILE").rstrip
 
     begin
-      update_eix if !FileUtils.uptodate?("/var/cache/eix", %w{/usr/bin/eix /usr/portage/metadata/timestamp})
+      update_eix if !FileUtils.uptodate?(eix_cachefile, %w{/usr/bin/eix /usr/portage/metadata/timestamp})
 
       search_output = nil
       Puppet::Util::Execution.withenv :LASTVERSION => version_format do
@@ -76,12 +77,13 @@
 
     version_format = "{last}<version>{}"
     search_format = "<category> <name> [<installedversions:LASTVERSION>] [<bestversion:LASTVERSION>] <homepage> <description>\n"
+    eix_cachefile = (eix "--print", "EIX_CACHEFILE").rstrip
 
     search_field = package_name.count('/') > 0 ? "--category-name" : "--name"
     search_value = package_name
 
     begin
-      update_eix if !FileUtils.uptodate?("/var/cache/eix", %w{/usr/bin/eix /usr/portage/metadata/timestamp})
+      update_eix if !FileUtils.uptodate?(eix_cachefile, %w{/usr/bin/eix /usr/portage/metadata/timestamp})
 
       search_output = nil
       Puppet::Util::Execution.withenv :LASTVERSION => version_format do
